lib/Sequence.fz

```diff
@@ -285,6 +285,18 @@ Sequence<T> ref is
   zip<U,V>(b Sequence<U>, f (T,U)->V) => asList.zip0 b.asList f
 
 
+  # the nth element in the sequence,
+  # starting at 1
```

Comment:
I would like to be consistent in starting to count elements at `0`. Then this feature would behave similar to `array.index [ ]`.

Reply:
@fridis yes, good point